Lib/test/test_calendar.py

```diff
@@ -619,6 +619,13 @@ def test_itermonthdays2(self):
                 self.assertEqual(days[0][1], firstweekday)
                 self.assertEqual(days[-1][1], (firstweekday - 1) % 7)
 
+    def test_iterweekdays(self):
+        for firstweekday in range(7):
+            cal = calendar.Calendar(firstweekday)
+            week = list(cal.iterweekdays())
+            self.assertEqual(week[0], firstweekday)
+            self.assertEqual(week[-1], (firstweekday - 1) % 7)
```

Comment:
I would compare ``week`` with the exact array it should be. You could use something like duque.rotate(), or write your own helper to rotate a list.